docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.4'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - database
+      - redis
+    ports:
+      - "3000:3000"
+    volumes:
+      - .:/app
+      - gem_cache:/usr/local/bundle/gems
+    env_file: .env
+    environment:
+      RAILS_ENV: development
+    entrypoint: ./entrypoints/docker-entrypoint.sh
+
+  database:
+    image: postgres:12.2
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: <%= ENV['DATABASE_PASSWORD'] %>
```

Comment:
does it .erb file?
see this article about how to pass env variables to docker-compose file
https://medium.com/better-programming/using-variables-in-docker-compose-265a604c2006